google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/MaterializedViewDefinition.java

```diff
@@ -57,6 +57,24 @@
     @Override
     public abstract Builder setType(Type type);
 
+    /**
+     * Sets the time partitioning configuration for the table. If not set, the table is not
```

Comment:
nit: do we want to update the comments here and elsewhere to indicate materialized view, rather than table?

Reply:
Yes -- Updated to `materialized view`.